django/contrib/admin/options.py

```diff
@@ -850,9 +850,12 @@ def action_checkbox(self, obj):
     def _get_base_actions(self):
         """Return the list of actions, prior to any request-based filtering."""
         actions = []
+        admin_action_names = {self.get_action(action)[1] for action in self.actions or []}
```

Comment:
`self.actions` already contains action names so we don't need to wrap them in `get_action()`. Moreover for a nonexistent name it raises `TypeError: 'NoneType' object is not subscriptable`. We've already ignored them in `filter(None, actions)` so it's safe to use `self.actions`.

I will adjust this.

Reply:
But wouldn't it failed if the `ModelAdmin`-specific action is defined by passing the function itself (instead of the function name), like the example written here: https://docs.djangoproject.com/en/3.0/ref/contrib/admin/actions/#adding-actions-to-the-modeladmin. I will write an additional test for this.

Reply:
Good point.